mongo/options/encryptoptions.go

```diff
@@ -10,11 +10,21 @@ import (
 	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
+// QueryType describes the type of query the result of Encrypt is used for.
+type QueryType int
+
+// These constants specify valid values for QueryType
+const (
+	QueryTypeEquality QueryType = 1
```

Comment:
```suggestion
	QueryTypeEquality QueryType = iota
```

[`iota`](https://github.com/golang/go/wiki/Iota) will help increment futures additions to this type.

Reply:
I want the value to be equal to the value defined in libmongocrypt: https://github.com/mongodb/libmongocrypt/blob/master/src/mongocrypt.h.in#L1506

It reduces risk that the wrong integer is passed through to libmongocrypt.